src/plugin/index.ts

```diff
@@ -75,8 +75,22 @@ export type ClientPluginIntegrationMap = ClientPluginIntegration & {
   placement: 'map';
 };
 
+export type FooterPlacementOrientation = 'left' | 'center' | 'right';
```

Comment:
There is no `center` insertion placement in the `Footer` component available. I think it can be removed here.